### ISSUE TITLE
Control Flow Guard targets in PE files are not correctly recognized as functions

### ISSUE THREAD

**Issue by marpie** [NationalSecurityAgency/ghidra#1547]
**Describe the bug**

Taking an arbitrary CFG aware CFG PE file (e.g. `mprddm.dll` - MD5: `94a6f38774655ef5c839e9906d1193f1`) from a recent Windows 10 installation, the functions in the CFG tables (e.g. GuardCFFunctionTable) are not marked as functions and are not disassembled automatically (see the screenshots below).

**To Reproduce**

1. Import `mprddm.dll`
2. Auto-analyze the file
3. Go to `GuardCFFunctionTable`
4. Select e.g. `IfObjectGetStatistics`

**Expected behavior**

It is expected that all entries in the CFG tables (`GuardCFFunctionTable`, `GuardAddressTakenIatEntryTable`, and `CFGuardLongJumpTarget`) should be recognized as code and disassembled correctly.

**Screenshots**

*GuardCFFunctionTable table*
![GuardCFFunctionTable table](https://user-images.githubusercontent.com/703785/74821439-e75f2100-5303-11ea-8d6d-d981037d87c3.png)

*IfObjectGetStatistics not recognized*
![IfObjectGetStatistics not recognized](https://user-images.githubusercontent.com/703785/74821507-0493ef80-5304-11ea-84a0-44e67cf4df1f.png)

*IfObjectGetStatistics expected behavior*
![IfObjectGetStatistics expected behavior](https://user-images.githubusercontent.com/703785/74821568-1ecdcd80-5304-11ea-80d3-396c497e69ae.png)

**Environment (please complete the following information):**
 - OS: Windows 10
 - Java Version: 11
 - Ghidra Version: 9.2 (master - `1c415bc053a`)

**Additional context**

The code at `Ghidra/Features/Base/src/main/java/ghidra/app/util/bin/format/pe/ControlFlowGuard.java` should be changed to mark the targets as functions and disassemble them accordingly.

**Comment by astrelsky** [comment]
This should be relatively simple to correct. The following else condition could be added after this if statement. https://github.com/NationalSecurityAgency/ghidra/blob/9faf39baed52fa41b6e9a3d0de4741477f5977f3/Ghidra/Features/Base/src/main/java/ghidra/app/util/bin/format/pe/ControlFlowGuard.java#L161-L164

```
else {
    Function f = program.getListing().getFunctionAt((Address) d.getValue());
    if (f == null) {
        program.getListing().createFunction((Address) d.getValue());
    }
}
```
Or something similar. I can put in a pr to fix this later today if nobody else does so by the time I get around to it.

**Comment by marpie** [comment]
I added a few modifications to better support CFG. I'm currently testing it with different binaries (DLLs and drivers) to make sure nothing breaks. I think I'll post my solution tomorrow if everything works out.

Adding `createFunction(...)` did not fix the problem entirely, but adding a disassemble before yields good results.

**Comment by astrelsky** [comment]
> I added a few modifications to better support CFG. I'm currently testing it with different binaries (DLLs and drivers) to make sure nothing breaks. I think I'll post my solution tomorrow if everything works out.
> 
> Adding `createFunction(...)` did not fix the problem entirely, but adding a disassemble before yields good results.

I see, I always forget about checking if it is disassembled first. What about using CreateFunctionCmd? I know that the ElfLoadHelper has a createOneByteFunction method but it appears there is no equivalend for Pe.

**Comment by marpie** [comment]
That was exactly my first try to use the DisassembleCmd/CreateFunctionCmd in the loader, but that would trigger it directly and would mess up PDB loading etc.

So I opted for creating a array and an Analyzer (see PR https://github.com/NationalSecurityAgency/ghidra/pull/1565).

I'm having problems utilizing the structure correctly as I have no idea how to access it later. So I opted to getInt() from the start, as that is enough for this case, but not a really clean way.
